catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/FilterReader.java

```diff
@@ -50,6 +46,9 @@ public FilterReader() throws JAXBException {
   @SuppressWarnings("unchecked")
   private <T> JAXBElement<T> unmarshal(InputStream inputStream, Class<T> tClass)
       throws JAXBException {
+    SAXParserFactory factory = XMLUtils.getInstance().getSecureSAXParserFactory();
+    factory.setNamespaceAware(true);
```

Comment:
It's not fair to assume the factory is thread safe, so this gets moved into the function and created on every `unmarshal` call.